tests/test_cli.py

```diff
@@ -0,0 +1,85 @@
+import click
+from click.testing import CliRunner
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) recommends a blank line between third-party imports and local imports.
```suggestion
from click.testing import CliRunner

```